dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/NormalizationTest.java

```diff
@@ -105,10 +104,8 @@ public void dimensionTest() throws DynatraceExporterException {
   @Test
   public void dimensionKeyTest() throws DynatraceExporterException {
     assertThrows(DynatraceExporterException.class, () -> MetricAdapter.toMintDimension(".", "b"));
-    MetricAdapter.toMintDimensionKey("test..e12");
     assertThrows(DynatraceExporterException.class, () -> MetricAdapter.toMintDimension(".a", "b"));
     assertThrows(DynatraceExporterException.class, () -> MetricAdapter.toMintDimension("a.", "b"));
-    MetricAdapter.toMintDimension("a..b", "b");
```

Comment:
Why were these removed? I think they're supposed to test that these _don't_ throw. Verifying the return value would be even better, of course.

Reply:
the update to normalization for the metrics in java is still outstanding. Some normalization is already done, but i am not sure if the normalization step follows all the latest guidelines for sanitization. I added the tests back, for now, but we will have to address the normalization step at some point in the future, possibly when adding tags and prefix options.